src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -130,11 +137,30 @@ trait SharedFileSystem {
       WdlFile(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
+    def localizeInputToCallDirectory(path: String): Path = {
+      val originalPath = Paths.get(path)
+      val callPath = Try {
+        val callInputsDirectory = LocalBackend.hostCallPath(workflowDescriptor, call.name, callKey.index).resolve("inputs")
+        if (!callInputsDirectory.toFile.exists) callInputsDirectory.toFile.mkdirs
+        val hash = hashPath(originalPath)
+        callInputsDirectory.resolve(s"$hash-${originalPath.getFileName.toString}")
```

Comment:
Just so I'm understanding, you're leery of the case where I have a task which takes as input files wiht the same names from previous tasks?